### PR TITLE
fix(XamlReader): Fix invalid Windows.UI lookups

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Using the `Color` type is now properly supported for the WinUI targets.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ internal static class XamlConstants
 		public const string BundleResourcePrefix = "ms-appx:///";
 
 		public const string RootFoundationNamespace = "Windows.Foundation";
-		public const string RootWUINamespace = "Windows.UI";
+		public const string RootWUINamespace = "Windows" + ".UI"; // Keep split for the WinUI conversion tool
 		public const string RootMUINamespace = "Windows.UI";
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs:16]: The name for this one is quite confusing.

`MUI` seems to mean "Microsoft.UI", but that's not the case?

File: src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs [hunk 2]
```diff
@@ -12,7 +12,7 @@ internal static class XamlConstants
 		public const string BundleResourcePrefix = "ms-appx:///";
 
 		public const string RootFoundationNamespace = "Windows.Foundation";
-		public const string RootWUINamespace = "Windows.UI";
+		public const string RootWUINamespace = "Windows" + ".UI"; // Keep split for the WinUI conversion tool
```
Comment (Youssef1313) [inline src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs:15]: This one won't get replaced by the tool, and the below will be replaced, right?

Just to make sure I understand the change here.

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
We have:

```csharp
public const string Text = RootWUINamespace + ".Text";
```


Before this change, this constant would have referred to `Microsoft.UI.Text` in WinUI flavor.

Now, this is only `Windows.UI.Text`. Is that intended?

File: src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs (hunk 2 above)
Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Markup/Reader/XamlConstants.cs:15]: That's correct. It's a mirror of this file: https://github.com/unoplatform/uno/blob/927a09f70d90f6fb1c9f03a3b77f5577d57240fe/src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlConstants.cs

**Review by MartinZikmund — APPROVED** [review]

**Review by carldebilly — APPROVED** [review]

**Pull request merged**